adal/tests/Test.ADAL.NET.Unit/DeviceCodeFlowTests.cs

```diff
@@ -135,14 +135,14 @@ public async Task AdfsPositiveTestAsync()
 
             TokenCache cache = new TokenCache();
             AuthenticationContext ctx = new AuthenticationContext(TestConstants.DefaultAdfsAuthorityTenant, false, cache);
-            AuthenticationResult result = await ctx.AcquireTokenByDeviceCodeAsync(dcr);
+            AuthenticationResult result = await ctx.AcquireTokenByDeviceCodeAsync(dcr).ConfigureAwait(false);
             Assert.IsNotNull(result);
             Assert.AreEqual("some-access-token", result.AccessToken);
             Assert.AreEqual(AdalHttpMessageHandlerFactory.MockHandlersCount(), 0);
         }
 
         [TestMethod]
-        public async Task AdfsPostMethodTest()
+        public async Task AdfsPostMethodTestAsync()
```

Comment:
I fixed these just now because the GitHub build status was red.

Reply:
what did you fix @bgavrilMS : I have not seen any other commit than @MarkZuber's

Reply:
@jmprieur - the changes in this file were fixing a build break. This is because we now enforce static analysis on unit test projects as well.